src/Sentry.AspNetCore/SentryTracingMiddleware.cs

```diff
@@ -35,74 +38,102 @@ internal class SentryTracingMiddleware
                 return null;
             }
 
-            _options.DiagnosticLogger?.LogInfo("Received Sentry trace header: {0}", value);
+            _options.DiagnosticLogger?.LogInfo("Received Sentry trace header '{0}'.", value);
```

Comment:
I wonder if debug is better here. It'll print out for all requests if it's properly integrated with a front end

```suggestion
            _options.DiagnosticLogger?.LogDebug("Received Sentry trace header '{0}'.", value);
```